help-beta/astro.config.mjs

```diff
@@ -8,6 +8,340 @@ export default defineConfig({
+						'spoilers',
+						'me-action-messages',
+						'create-a-poll',
+						'collaborative-to-do-lists',
+						'global-times',
+						'start-a-call'
+					]
+				},
+				{
+					label: 'Sending messages',
+					items: [
+						'open-the-compose-box',
+						'mastering-the-compose-box',
+						'resize-the-compose-box',
+						'typing-notifications',
+						'preview-your-message-before-sending',
+						'verify-your-message-was-successfully-sent',
+						'edit-a-message',
+						'delete-a-message',
+						'view-and-edit-your-message-drafts',
+						'schedule-a-message',
+						'message-a-channel-by-email'
+					]
+				},
+				{
+					label: 'Reading messages',
+					items: [
+						'reading-strategies',
+						'inbox',
+						'recent-conversations',
+						'combined-feed',
+						'channel-feed',
+						'left-sidebar',
+						'message-actions',
+						'marking-messages-as-read',
+						'marking-messages-as-unread',
+						'configure-unread-message-counters',
+						'emoji-reactions',
+						'view-your-mentions',
+						'star-a-message',
+						'view-images-and-videos',
+						'view-messages-sent-by-a-user',
+						'link-to-a-message-or-conversation',
+						'search-for-messages',
+						'printing-messages',
+						'view-the-markdown-source-of-a-message',
+						'view-the-exact-time-a-message-was-sent',
+						'view-a-messages-edit-history',
+						'collapse-a-message',
+						'read-receipts'
+					]
+				},
+				{
+					label: 'People',
+					items: [
+						'user-list',
+						'status-and-availability',
+						'user-cards',
+						'view-someones-profile',
+						'direct-messages',
+						'user-groups',
+						'find-administrators',
+					]
+				},
+				{
+					label: 'Channels',
+					items: [
+						'introduction-to-channels',
+						'create-a-channel',
+						'pin-a-channel',
+						'change-the-color-of-a-channel',
+						'unsubscribe-from-a-channel',
+						'manage-inactive-channels',
+						'move-content-to-another-channel',
+						'view-channel-subscribers',
+					]
+				},
+				{
+					label: 'Topics',
+					items: [
+						'introduction-to-topics',
+						'rename-a-topic',
+						'resolve-a-topic',
+						'move-content-to-another-topic',
+						'delete-a-topic',
+					]
+				},
+				{
+					label: 'Notifications',
+					items: [
+						'channel-notifications',
+						'topic-notifications',
+						'follow-a-topic',
+						'dm-mention-alert-notifications',
+						'mute-a-channel',
+						'mute-a-topic',
+						'mute-a-user',
+						'email-notifications',
+						'desktop-notifications',
+						'mobile-notifications',
+						'do-not-disturb',
+					]
+				},
+				{
+					label: 'Apps',
+					items: [
+						{ 
+							'label': 'Download apps for every platform',
+							link: 'https://zulip.com/apps/'
+						},
+						'desktop-app-install-guide',
+						'supported-browsers',
+						'configure-how-links-open',
+						'connect-through-a-proxy',
+						'custom-certificates',
+					]
+				},
+				{
+					label: 'Zulip Administration',
```

Comment:
I don't think the double-nesting looks very good.

I think what we want instead is something more like what we have in `main`, where the administration heading is just a larger heading without collapsing. But maybe that'd hard to do. 

![image](https://github.com/user-attachments/assets/479fa1a8-6a69-405e-ab2d-fc57f5ee8413)

Reply:
It's not hard to do, the solution is a bit hacky, but I think it's workable enough for now. I've pushed the relevant change. I've set the font-size to 1.15 rem since that what looked good to me when testing manually, but we can focus on the exact size later on during the design pass. I've used rem since astro/starlight also uses rem for their sizing.

<img width="308" alt="Screenshot 2024-08-26 at 11 18 23 AM" src="https://github.com/user-attachments/assets/bc0ed7d6-d4b8-4b27-816c-a4adc2bf7903">

Reply:
We may later want to do a design pass on that, yeah. I'm not sure we want to keep that heading vs. finding some other way to visually separate different sub-areas of the sidebar.

Reply:
I think that might be better left to a later design pass.